lib/rspecq/queue.rb

```diff
@@ -241,11 +249,36 @@ def flaky_jobs
       requeued - @redis.hkeys(key_failures)
     end
 
+    def fail_fast
+      return nil unless published?
+
+      raw_value = @redis.hget(key_queue_config, 'fail_fast')
+      @fail_fast ||= raw_value.nil? ? nil : raw_value.to_i
```

Comment:
Can we avoid memoization here to make this thing simpler? How often is this method going to be called? Also, this has no benefit because you anyway call `hget` above.

Reply:
Actually, I see that this method is only used for reporting reasons. In that case, I don't believe it's worth it at all. Let's just do in the reporter (approximately):

> Build failing fast after N failures...